en/includes/guides/organization-management/invite-parent-organization-users.md

```diff
@@ -63,3 +63,73 @@ Inorder to view or delete the invitations via {{ product_name }} Console:
     ![View invitations]({{base_path}}/assets/img/guides/organization/manage-organizations/view-invitations.png){: width="600" style="display: block; margin: 0;"}
 
 4. If required, invitations can be deleted by clicking the **Delete** icon.
+
+## Invite users without sending email notifications from {{product_name}}
+
+{% if manageNotificationsInternallyPropertySupportedUpdateLevel %}
+
+!!! note
+    This support is available starting from update level {{ manageNotificationsInternallyPropertySupportedUpdateLevel }}.
+
+{% endif %}
+
+In order to disable sending email notifications internally from {{ product_name }}, `manageNotificationsInternally` 
+property can be set to false in Parent Organization's User Invitation API. Next, the following sample request can be 
+used to obtain the confirmation code without sending email notifications.
+
+   ``` bash
+   curl --location --request POST 'https://{{ host_name }}/t/carbon.super/o/api/server/v1/guests/invite' \
```

Comment:
/t/carbon.super part should be come from a placeholder value for IS docs 

/t/root org name tor Asgardeo docs

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4832/commits/e0e54cd7f4e226e2989f339bae4a6918de0725f6